component/Page/Profile/index.tsx

```diff
@@ -172,7 +180,8 @@ export default function ProfilePage(props: PageProps) {
                                     </div>
                                 </div>
                                 <div className="flex flex-row gap-2 items-center">
-                                    <button className="border border-gray02 h-10 w-10 rounded-lg flex justify-center items-center" onClick={() => showDownloadToast()}>
+                                    <button className="border border-gray02 h-10 w-10 rounded-lg flex justify-center items-center" 
+                                        onClick={() => showDownloadToast()}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `showDownloadToast` function is called without an event name. If you want to track this event, you should pass the appropriate event from the `BetterSocialEventTracking` enum.
```diff
-    onClick={() => showDownloadToast()}>
+    onClick={() => showDownloadToast(BetterSocialEventTracking.YOUR_EVENT_NAME)}>
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->